application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -184,11 +186,10 @@
 
   $(document).ready(function() {
     if ($('#healthCheck').length) {
-      let jobState = $('input[name="jobState"]').val();
+      let jobState = $('jobState').attr('data-state')
```

Comment:
Again, I don't see the need for a custom HTML element (which is not event properly defined).